src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -43,6 +43,7 @@
   <PropertyGroup Condition="'$(TestWithCore)' != 'true'">
     <XunitExecutable Condition="'$(XunitExecutable)' == ''">xunit.console.exe</XunitExecutable>
     <DebugTestFrameworkFolder>dnx45</DebugTestFrameworkFolder>
+    <DelaySign Condition="'$(OSEnvironment)' == 'Windows_NT'">false</DelaySign>
```

Comment:
I don't want this to be specific here. I want us to more generally support this in sign.targets